### PR TITLE
Update matmut.sk

### PR DESCRIPTION
**Pull request by MrKrakenied**

### CODE REVIEW

**Comment by PinkLolicorn** [comment]
Please use spaces instead of tabs, do not reformat whole file. This is real enterprise grade project. No jokes.

File: plugins/Skript/scripts/matmut.sk [hunk 1]
```diff
@@ -3,25 +3,27 @@
...
 # {{ u can join us D: }}
 #
 
-function matgen(n: number) :: objects: 
+
+function matgen(n: number) :: vectors: 
     # send "matgen started, _n = %{_n}%" to console
 
-    set {_tmp} to (1.0 / {_n}) / {_n}
+    set {_tmp} to 1.0 / {_n} / {_n}
```
Comment (dzikoysk) [inline plugins/Skript/scripts/matmut.sk]: Skript can't into math, so it's important to wrap operations into sections like it was before  `(1.0 / {_n}) / {_n}`

~ Source:
obraz

File: plugins/Skript/scripts/matmut.sk [hunk 2]
```diff
@@ -3,25 +3,27 @@
...
             set {_index_1} to (loop-number-1) - 1.0
             set {_index_2} to (loop-number-2) - 1.0
-            #send "index 1, index 2 = %{_index_1}%, %{_index_2}%" to console
-
-            set {_a::%{_index_1}%::%{_index_2}%} to (({_tmp} * 1.0) * ({_index_1} - {_index_2})) * ({_index_1} + {_index_2})
-            send "_val = %{_a::%{_index_1}%::%{_index_2}%} * 1.0%" to console
-
+            # send "index 1, index 2 = %{_index_1}%, %{_index_2}%" to console
+            
+            set {_a::%{_index_1}%n%{_index_2}%} to (({_tmp} * 1.0) * ({_index_1} - {_index_2})) * ({_index_1} + {_index_2})
```
Comment (dzikoysk) [inline plugins/Skript/scripts/matmut.sk]: Why did you remove a two-dimensional variable?

File: plugins/Skript/scripts/matmut.sk [hunk 3]
```diff
@@ -3,25 +3,27 @@
...
+            # send "_val = %{_a::%{_index_1}%::%{_index_2}%} * 1.0%" to console
```
Comment (PinkLolicorn) [inline plugins/Skript/scripts/matmut.sk]: Please do not hide debugging messages, the _val calculation is important and should be visible.

File: plugins/Skript/scripts/matmut.sk [hunk 4]
```diff
@@ -3,25 +3,27 @@
...
         loop {_n} times:
             set {_index_1} to (loop-number-1) - 1.0
             set {_index_2} to (loop-number-2) - 1.0
-            #send "index 1, index 2 = %{_index_1}%, %{_index_2}%" to console
-
-            set {_a::%{_index_1}%::%{_index_2}%} to (({_tmp} * 1.0) * ({_index_1} - {_index_2})) * ({_index_1} + {_index_2})
-            send "_val = %{_a::%{_index_1}%::%{_index_2}%} * 1.0%" to console
-
+            # send "index 1, index 2 = %{_index_1}%, %{_index_2}%" to console
+
```
Comment (PinkLolicorn) [inline plugins/Skript/scripts/matmut.sk]: You should leave empty lines empty, not filled with spaces or any other whitespace characters.

**Comment by MrKrakenied** [comment]
We can't return {x::x::x}, only {x::x}

File: plugins/Skript/scripts/matmut.sk [hunk 5]
```diff
@@ -3,11 +3,13 @@
...
 # - Dzikoysk (https://github.com/dzikoysk)
 # - Michał Kubik (https://github.com/bopke)
+# - MrKrakenied (https://github.com/MrKrakenied)
 #
 # {{ u can join us D: }}
 #
 
-function matgen(n: number) :: objects: 
+
+function matgen(n: number) :: vectors:
```
Comment (dzikoysk) [inline plugins/Skript/scripts/matmut.sk]: I'm not sure that vector is the proper return type.

~ https://skriptlang.github.io/Skript/classes.html#vector
